### DISCUSSION TITLE
aframe + arjs: avoid body styles / specify container

### DISCUSSION THREAD

**Discussion by hatsumatsu** [AR-js-org/AR.js#258] [Q&A]
Following the aframe examples ARjs adds inline styles to the `<body>` element which makes it hard top embed the ar scene in an existing application. This also defies the idea of the`embedded` attribute of aframe scenes.

Is there any way to define a container element that is not the document's `<body>` ?

PS: On a side note I noticed that aframe-ar.js adds the video and adds inline styles when the script is present no matter if there are any marker entities present in the scene:

**Comment by kalwalt** [comment]
@hatsumatsu I think we should let the dev more freedom to choose which container element to use. Maybe with a sort of a callback or code injection?
